discord.sh

```diff
@@ -5,6 +5,7 @@ socat $SOCAT_ARGS \
     UNIX-CONNECT:$XDG_RUNTIME_DIR/discord-ipc-0 \
     &
 socat_pid=$!
+disable-breaking-updates.py &
```

Comment:
Why is this launched in background?

Reply:
Doing this synchronously (not in the background) will make it less likely to break Discord, if Discord does not rewrite `settings.json` atomically (which it really should, but it might not, because atomic overwrites are not straightforward on Windows).